crypto/build.gradle.kts

```diff
@@ -15,6 +15,7 @@ dependencies {
   implementation("org.bouncycastle:bcprov-jdk15on:1.70")
   implementation("org.bouncycastle:bcpkix-jdk15on:1.70")
   implementation(project(":common"))
+  testImplementation(kotlin("test"))
```

Comment:
mostly unrelated to this PR (esp. because i'm using this in `dids` as well), should we use junit5 instead?